Homeowrk4.ipynb

```diff
@@ -0,0 +1,281 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Assignment 1\n",
+    "\n",
+    "Creat a class that represents a textfile (e.g. called `Text`). The class should have the following properties:\n",
+    "\n",
+    "- a variable to hold the path to the file\n",
+    "- a variable that holds the type of file (e.g. plaintext, pdf, image)\n",
+    "- a variable that holds the filesize\n",
+    "- a variable `text` that holds the text content of a file\n",
+    "- a method to load the text content of a file into the `text` variable\n",
+    "- a method that sets the filesize property of an object according the size of the file at `filepath`\n",
+    "- a method that accepts another text object and returns true if that text object is smaller in size than the current one; otherwise false\n",
+    "\n",
+    "Then create three Text objects and put them in a list. For each element in the list print filepath, type of file, and filesize.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "loading/path/to/my/file\n",
+      "pdf\n",
+      "30\n",
+      "loading/path/to/my/file\n",
+      "txt\n",
+      "15\n",
+      "loading/path/to/my/file\n",
+      "doc\n",
+      "45\n"
+     ]
+    }
+   ],
+   "source": [
+    "# your code\n",
+    "class Text:\n",
+    "    def __init__(self, filepath, filetype, filesize):\n",
+    "        self.path_to_file=filepath\n",
+    "        self.filetype=filetype\n",
+    "        self.filesize=filesize\n",
+    "    def load_text(self):\n",
+    "        print(\"loading\" + self.path_to_file) \n",
+    "    def smaller_size(filesize, current_size):\n",
+    "        if filesize < current_size:\n",
+    "            return True\n",
+    "        return False\n",
+    "\n",
+    "t1=Text(\"/path/to/my/file\", \"pdf\", 30)\n",
+    "t1.load_text()\n",
+    "print(t1.filetype)\n",
+    "print(t1.filesize)\n",
+    "\n",
+    "t2=Text(\"/path/to/my/file\", \"txt\", 15)\n",
+    "t2.load_text()\n",
+    "print(t2.filetype)\n",
+    "print(t2.filesize)\n",
+    "\n",
+    "t3=Text(\"/path/to/my/file\", \"doc\", 45)\n",
+    "t3.load_text()\n",
+    "print(t3.filetype)\n",
+    "print(t3.filesize)    "
```

Comment:
The assignment asks to put the text objects "in a list. For each element in the list print filepath, type of file, and filesize."